pkg/roachprod/cloud/gc_aws.go

```diff
@@ -316,3 +325,105 @@ func GCAWSKeyPairs(l *logger.Logger, dryrun bool) error {
 	}
 	return nil
 }
+
+// gc garbage-collects expired clusters, unused SSH key pairs in a
+// single AWS account
+func gc(l *logger.Logger, dryrun bool) error {
+	var (
+		wg            sync.WaitGroup
+		keyPairErr    error
+		combinedError error
+	)
+
+	wg.Add(1)
```

Comment:
Nit: since there is only one goroutine now, it's kind of awkward to have to use a semaphore. Instead, you could use `errgroup.Group`, which hides it. (It uses `sync.WaitGroup` internally.)

Reply:
Or send the error through a channel:

```go
go func() {
    keyPairErr <- gcAWSKeyPairs()
}()

// ...

err = errors.CombineErrors(err, <-keyPairErr)
```